### DISCUSSION TITLE
Help with Power Settings

### DISCUSSION THREAD

**Discussion by sherpascott** [seerge/g-helper#3096] [Q&A]
Hey I just wanted some help with the custom power settings. I'm a noob at all this and I'm scared I'm screwing things up, so I came here to see if anyone could help me.

I set my cpu to a lower wattage in "silent" mode (6 watts) with a max of 10 w.  However, when I look at g helper, it shows discharging 15 watts sometimes. Did I mess it up, or is that because it's also supplying power to my screen and stuff?

**Comment by sherpascott** [comment]
![image](https://github.com/user-attachments/assets/c24e58fc-6614-444a-afa2-6865ddededf6)
![image](https://github.com/user-attachments/assets/e63a2934-7534-4204-855f-32671426bf13)

**Comment by sherpascott** [comment]
That is a poor example but the Discharging number sometimes goes all the way up to 20W, and likes to sit a little higher than the cap I set

**Comment by seerge** [comment]
@sherpascott hello,

This is not an "issue" with an app, but rather a question. Therefore I'm moving this to discussions.

To answer the question - it's absolutely normal that battery power drain is higher than CPU limits (12W) that you have set as you have screen / motherboard / ssd and lots of other stuff that uses power as well.

If you want to minimize power drain on battery - make sure to turn on Battery Saver mode in Windows (As it stops lots of background tasks system runs) and also check Task Manager for some random things running as well.

Thanks
